packages/kit/src/core/adapt/prerender.js

```diff
@@ -5,6 +5,12 @@ import { mkdirp } from '../filesystem/index.js';
 import { __fetch_polyfill } from '../../install-fetch.js';
 import { SVELTE_KIT } from '../constants.js';
 
+/**
+ * @typedef {import('types/config').PrerenderErrorHandler} PrerenderErrorHandler
+ * @typedef {import('types/config').ValidatedConfig['kit']['prerender']['onError']} OnErrorConfig
+ * @typedef {import('types/internal').Logger} Logger
+ */
+
```

Comment:
Logger is used twice, PrerenderErrorHandler is used a few times, and OnErrorConfig is just a really long line which muddied up the definition it was being used in.